fs_mark/test_armv7a7-imx6ull-evk.yaml

```diff
@@ -0,0 +1,131 @@
+# This yaml contain armv7a7-imx6ull-evk specific fs_mark tests
+test:
+  type: harness
+  harness: fs_mark.py
+
+  targets:
+    value: [armv7a7-imx6ull-evk]
+
+# TODO: full fs filling tests
```

Comment:
Are there any specific problems with such types of test? If there are any issues, it might be worth mentioning them. If not, I would leave it as it is

Reply:
Is it already added?